code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -391,8 +391,9 @@ This is here to make the tiles around the station mininuke change when it's arme
 
 /obj/machinery/nuclearbomb/selfdestruct/proc/SetTurfs()
 	if(loc == initial(loc))
-		for(var/turf/simulated/floor/bluegrid/T in orange(src, 1))
-			T.icon_state = (timing ? "rcircuitanim" : "bcircuit")
+		for(var/N in nuke_tiles)
+			var/turf/T = N
+			T.icon_state = (timing ? "rcircuitanim" : initial(T.icon_state))

```

Comment:
this breaks tiles with varedited `icon_state`s. try setting it to `icon_regular_floor` instead